redeem/gcodes/G29.py

```diff
@@ -104,11 +106,12 @@ def execute(self, g):
         for i in range(len(probes)):
             gcodes += "    G30 P{} S F{}; Probe point {}\n".format(i, probe_speed, i)
         gcodes += "    G31 ; Dock probe\n"
-        gcodes += "    M561 U; (RFS) Update the matrix based on probe data\n"
-        gcodes += "    M561 S; Show the current matrix\n"
+        if bed_comp:
+            gcodes += "    M561 U; (RFS) Update the matrix based on probe data\n"
+            gcodes += "    M561 S; Show the current matrix\n"
         gcodes += "    M500; (RFS) Save data\n"
 
-        # REset probe points
+        # Reset probe points
         self.printer.probe_points = []
         self.printer.config.set("Macros", "G29", gcodes)
```

Comment:
i think this means that if i had written a custom G29 and then executed G29.1, I'd lose my macro? maybe we should make a note in the description that executing 29.1/2 will over write the G29 macro in the local.cfg (?)?